MODULE.bazel

```diff
@@ -15,6 +15,13 @@ bazel_dep(
     version = "2.4.0",
     repo_name = "build_bazel_rules_swift",
 )
+
+# Test with latest rules_swift without requiring users to use this version
+single_version_override(
```

Comment:
are you going to remove this before landing? and do we need to bump the rules_swift minimum?

Reply:
Idea is to test with this version in ci so we override it here. We don't need to update the minimum here since we support 2.x and 3.x